nextflow_schema.json

```diff
@@ -35,7 +35,7 @@
                 "outdir": {
                     "type": "string",
                     "description": "The output directory where the results will be saved.",
-                    "default": "./results",
+                    "default": ".&#x2F;results",
```

Comment:
Is this a typo?

Reply:
ok I guess not since it is everywhere like this

Reply:
I think this was set by the nextflow schema web editor, I will fix it back to `/`, well spotted!